src/server/yarn.lock

```diff
@@ -1,2920 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Deleted because if we use npm inside container we better use it on host machine too